### ISSUE TITLE
Refute the open letter's claims of discrimination

### ISSUE THREAD

**Issue by CrazyPython** [rms-support-letter/rms-support-letter.github.io#2317]
Stallman asked women out in a way we consider creepy. The letter phrases this by saying Stallman has sexually harassed women for decades. The letter seeks to group him in the same category as gropers and actual rapists.
In 1993, he defined pedophilia slightly differently. He made a genuine apology. This is the basis of the "pedo apologist." https://stallman.org/archives/2019-jul-oct.html#14_September_2019_(Sex_between_an_adult_and_a_child_is_wrong)
After condemning the rape of a 17 year old girl in the Epstein case, Marvin Minksy had sex with the girl, unaware the girl was being coerced by Epstein. In accordance with the law of Canada and most U.S. states, it is legal to have sex with a 16 year old girl. Nonetheless, we believe these leaked private discussions make him equivalent to a rapist. https://geoff.greer.fm/2019/09/30/in-defense-of-richard-stallman/
The original open letter's Appendix has two true claims of sexual harassment:
- In 1999, he had a mattress in his office which he slept on. This made some women avoid RMS because they considered it abnormal and creepy.
- In 1985, he told a woman he'd kill himself if he didn't date her.
The "hot ladies" nametag was fake; it was vandalized from a third party. https://blog.dachary.org/2020/02/10/how-the-cancel-culture-was-leveraged-against-rms/#post-4297:~:text=As%20for%20the%20label%20on%20Richard,removed%20the%20ending%3A%20(also%3A%20hot%20ladies)
He has never been accused of groping or sexual assault.
Stallman has hundreds of posts on his personal blog condemning rape, and they cherry-pick the one post where he argued for a different definition of rape. https://web.archive.org/web/20210325013942/https://stallman.org/archives/2018-may-aug.html
The GNU Kind Communication Guidelines ask you to honor gender identity. The letter says it is "still transphobic" and accuses him of "thinly disguised transphobia," possibly because it includes calling someone via only gender-neutral pronouns such as "they."
---

I hastily wrote this. Still needs some copyediting. But the information is there.

**Comment by appetrosyan** [comment]
Alleging misconduct based on false information is grounds for a defamation lawsuit. If they include even one instance of a false or inflated accusation, they're on the hook. RMS isn't too Rich, but I'm sure that with the free will he has amassed here, raising the money to win the lawsuit wouldn't be too difficult.

**Comment by M-i-k-o-t-o** [comment]
There are two more issues about this

https://github.com/rms-support-letter/rms-support-letter.github.io/issues/1338
https://github.com/rms-support-letter/rms-support-letter.github.io/issues/1637

**Comment by 6r1d** [comment]
There is [yet another complaint](https://www.reddit.com/r/gnu/comments/mdbe09/a_support_letter_for_richard_stallman/gs8ujb8?utm_source=share&utm_medium=web2x&context=3) in my post about the support letter.

**Comment by Tw1ddle** [comment]
> Marvin Minksy had sex with the girl, unaware the girl was being coerced by Epstein.

Marvin Minsky's Wikipedia page says "There has been no allegation that sex between them took place nor a lawsuit against Minsky's estate." - https://en.m.wikipedia.org/w/index.php?title=Marvin_Minsky&oldid=1011820349#Association_with_Jeffrey_Epstein

**Comment by Hhina** [comment]
Or donʼt.

It is unfortunate that Stallman is being misrepresented, but **thatʼs not the point**.
The FSF should not elect people based on their popularity, or on their actions and opinions unrelated to free software.
To counter [the “open letter”](https://rms-open-letter.github.io/) on these grounds suggests that Richard Stallman is unfit for his position because he is *unfashionable*. That shouldnʼt be the case.

Iʼm not saying that defending people against slander or correcting misinformation are bad things to do.
But our focus should be in issues of policy, not of character.

**Comment by gorshunovr** [comment]
[Low grade "journalists" and internet mob attack RMS with lies. In-depth review.](https://sterling-archermedes.github.io/) - in-depth article from 2019 on the subject.

**Comment by pastk** [comment]
Some trans people do not agree with the accusation of RMS as a transphobic person: https://lwn.net/Articles/850530/

**Comment by Huy-Ngo** [comment]
I suggest adding these articles too:

- [Leah Rowe][leah]'s defense regarding the accusation of him being transphobic
- [We The Web][wtw]'s defense backed by feminist activist
- [Jorge Morais][jorge]'s defense regarding RMS defending Minsky.

[leah]: https://libreboot.org/news/rms.html
[wtw]: https://www.wetheweb.org/post/cancel-we-the-web
[jorge]: https://jorgemorais.gitlab.io/justice-for-rms/

**Comment by 6r1d** [comment]
> I suggest adding these articles too:

Thank you, the same articles from this issue, it's great to see them spreading! :-)

https://github.com/rms-support-letter/rms-support-letter.github.io/issues/245

**Comment by 6r1d** [comment]
We just had a long dialog in Discord about GCC and its fate is very concerning.
RMS is removed from GCC steering commitee. It might be [renamed soon](https://twitter.com/blelbach/status/1376904459444383745), as well.
I'm worried if the license will one day be changed, too, because it will influence a lot of projects from that point on.

There is some evidence from the other side of the argument, and I intend to learn from it, too: "[Remove RMS from the GCC Steering Committee](https://news.ycombinator.com/item?id=26598066)". Let it be an important lesson for us.

RMS did not accept an offer to integrate LLVM to GCC, which is regrettable. The main argument against him there is the fact he hadn't even seen the said proposal. And I don't know if he addressed it after, besides that letter ten years later. And to be honest, this is a sad fact:

> I am stunned to see that we had this offer.
> Now, based on hindsight, I wish we had accepted it.
>
> If I had seen it back then, I would not have had the benefit of
> hindsight, but it would clearly have been a real possibility.  Nothing
> would have ruled it out.
>
> I wish I had known about the offer.
>
> Oh well.

But besides that, there's a different side of the argument:
* concerns about different patches held privately and not being contributed back
* concerns about private builds with said private patches supporting different architectures

There is also an answer about Apple and other companies contributing to both. Then again, I expect some toolchains for embedded to only contain a modified GCC build for convenience.

So, RMS has a point about supporting compilation for as many architectures as possible, but the fact he ignored LLVM proposal for ten years is weird. Another weird thing is that he did not get spammed by many emails from FSF explaining how important LLVM would be, immediately. If I was in FSF, I'd spam him every day to no end about that proposal, it's that important. But I doubt it's possible anyone will agree to integrate LLVM now, especially considering the license.

The argument on The Hacker News is that:

> RMS's attitude is harming the project

It's more nuanced than that, though. Why wasn't he informed? Would he agree to integrate LLVM should he be informed? Would LLVM developers accept GNU GPL? These are the things we don't know.

**Comment by Tw1ddle** [comment]
@6r1d Keep in mind that the open letter to remove Stallman and the FSF board does not even mention this stuff. I think it is better to focus on refuting the claims in the letter.

Stallman is not perfect and has missed things and made some bad decisions on a technical level.

**Comment by 6r1d** [comment]
> Stallman is not perfect and has missed things and made some bad decisions on a technical level.

It's not about "not being perfect". I think we all admit he's human. It's about "what happened so nobody cared to tell him at all?!". And if he was actually told, what happened then? It's a very nuanced topic.

> I think it is better to focus on refuting the claims in the letter.

Alright, I'll move notes on GCC politics for staff discussion.

**Comment by Tw1ddle** [comment]
@6r1d Agreed, it seems odd to put all the blame on Stallman for failures of that kind. That the remove letter does not mention these sorts of reasons is interesting by itself... perhaps it just wasn't on their list of priorities.

**Comment by Sarthak2319** [comment]
https://en.wikipedia.org/w/index.php?title=Wikipedia:Biographies_of_living_persons/Noticeboard&diff=727727586&oldid=727703442

**Comment by 6r1d** [comment]
Citing a link above, which details an update at June 2016:

> Right now, the only way it could included would be to directly quote and attribute Stallman's blog, which really just leaves it hanging, so I think pedophile apologists would perceive it as confirming their biases and I truly think it would have more of backfire effect than anything else.

> I've just written to Stallman and he explained that his views have changed significantly since the time he wrote these things. I have changed my opinion on this. It wouldn't be fair to include this in the article. I would like to recommend that this be closed now.

**Comment by shenlebantongying** [comment]
Leah Rowe seems has been working on this page, and it looks pretty cool

https://stallmansupport.org/index.html

**Comment by 6r1d** [comment]
There's yet another article: "[A Critique of the Open Letter Calling for the Removal of RMS](https://meribold.org/2021/04/07/critique-of-rms-open-letter/#fnref:twitter-response-clarification)"
